### PR TITLE
add sentry error tracking to flask app

### PR DESCRIPTION
**Pull request by camallen**
add error tracking for the aggregation online endpoints - this will be helpful debugging the reducer / extractor errors in say caesar.

I haven't tested this locally to make sure the app boots correctly.

### CODE REVIEW

**Comment by CKrawczyk** [comment]
Travis is failing with `sentry_sdk.utils.BadDsn: Invalid project in DSN ('(None,)')`  I think it is because the `SENTRY_KEY` and `SENTRY_PROJECT_ID` env variables don't exist in Travis and defaults to `None`.

One way to get around this is to use a [`Mock`](https://docs.python.org/3/library/unittest.mock.html) for the `sentry_sdk.init` function in the [test_routes.py](https://github.com/zooniverse/aggregation-for-caesar/blob/master/panoptes_aggregation/tests/router_tests/test_routes.py) file.  I can help add this in if you want.

**Comment by camallen** [comment]
> Travis is failing with `sentry_sdk.utils.BadDsn: Invalid project in DSN ('(None,)')` I think it is because the `SENTRY_KEY` and `SENTRY_PROJECT_ID` env variables don't exist in Travis and defaults to `None`.
> 
> One way to get around this is to use a [`Mock`](https://docs.python.org/3/library/unittest.mock.html) for the `sentry_sdk.init` function in the [test_routes.py](https://github.com/zooniverse/aggregation-for-caesar/blob/master/panoptes_aggregation/tests/router_tests/test_routes.py) file. I can help add this in if you want.

I think the better way it to use the built in std behaviour https://docs.sentry.io/error-reporting/configuration/?platform=python#dsn

```
If this value is not provided, the SDK will try to read it from the SENTRY_DSN environment variable. If that variable also does not exist, the SDK will just not send any events.
```

File: setup.py [hunk 1]
```diff
@@ -72,6 +73,7 @@
             'coverage',
             'coveralls',
             'flake8',
+            'sentry-sdk[flask]'
```
Comment (CKrawczyk) [inline setup.py]: I don't think this needs to be here.  Requirements should only show up one time in the `setup.py` file.  The `test` section should only include the packages needed to run the tests/coverage/linting.

Comment (camallen) [reply setup.py]: thanks - removed it :)

**Pull request merged**